src/patient-registration/patient-registration.component.tsx

```diff
@@ -72,7 +72,7 @@ export const initialFormValues: FormValues = {
   deathCause: '',
 };
 
-export const getDeathInfo = (values: FormValues) => {
+export const getDeathInfo = values => {
```

Comment:
We should try have some form of typing here so that we have some idea what values is going to look like.

Reply:
We explained why we removed the type for the input in another comment earlier: https://github.com/openmrs/openmrs-esm-patient-registration/pull/45#discussion_r487118574

Short version: we aimed at testing this helper method and while doing so found out, that the object we get when submitting the form only contains string values (no values of type Date anymore like defined in `FormValues`). Would you have a solution for this in mind already?

Reply:
I may be answering an already answered question, but if the typing for `Date` in `FormValues` is not needed, can we not just change that to `string`? I think the `Date` typing would only be needed if we were doing parsing or manipulation to the `Date` input. If our API is expecting to receive a `string` for the Death Date, then our typings should reflect that

Reply:
Okay, so we changed the type of `deathDate` in `FormValues` from `Date` to `string`. This enabled us to use typing here.